docs/docs/reference/02-manual-installation-launch.md

```diff
@@ -0,0 +1,289 @@
+
+## Access the REST API
+
+Visit [localhost:5000](http://localhost:5000) to see the Swagger UI and documentation for the REST API.
+
+## Connect a terminal session to the container
+
+It is sometimes useful to connect to the application container while it is running to inspect files or run commands.
+This can be accomplished with the `docker container exec` command using the container name.
+
+:::note
+
+Change `CONTAINER_NAME` in the command below to the name of the running container.
+
+:::
+
+```bash
+docker container exec -it -u ros2 CONTAINER_NAME /bin/bash
+```
+
+The flags `-it -u ros2` tell docker to attach an interactive terminal as the `ros2` user. The command `/bin/bash`
+starts a shell process.
+
+:::tip
+
+You can find the names of all running containers with the `docker ps` command.
+
+Docker containers can also be given an explicit name when started with `docker run --name CONTAINER_NAME [...]`.
+
+:::
+
+Once attached, run any commands in the context of the container. For example, `ros2 topic list` will show the current
+ROS 2 topics.
+
+Detach from the container with CTRL+D or with the `exit` command.
+
+### Display sharing
+
+:::note
+
+Native display sharing for GUI applications is only supported on Linux hosts.
+
+:::
+
+To run GUI applications like RViz in the docker container, it is necessary to share additional environment variables
+for the display and X11 server.
+
+:::caution
+
+Running `xhost +` disables access control to the X11 server on the host, allowing the container to access the display.
+It also allows any other process to access the server, which carries security implications. Use with caution.
+
+Access control can be re-enabled with `xhost -`.
+
+:::
+
+```bash
+xhost +
+docker container exec -it -u ros2 -e DISPLAY="$DISPLAY" -e XAUTHORITY="$XAUTH" CONTAINER_NAME /bin/bash
+```
+
+You should then be able to run `rviz2` inside the container and see the window appear.
+
+## Persistent user data
+
+AICA applications and URDF hardware can be uploaded to a user database through the API or Developer Interface UI.
+Because the docker container is isolated from the host filesystem, the local database will be lost when the container
+exists. To persist local data between sessions, create a dedicated directory somewhere on the host. For example,
+use `mkdir ~/.aica-data` to keep the data folder hidden in the home folder. Then execute the normal run command with an
+additional volume mount for the user data.
+
+:::note
+
+Change `/path/to/data` in the command below to a desired location for the data (e.g., `~/.aica-data` or elsewhere)
+
+:::
+
+<Tabs groupId="os">
+<TabItem value="linux" label="Linux">
+
+```bash
+docker run -it --rm \
+  --privileged \
+  --net=host \
+  -v /path/to/license:/license:ro \
+  #highlight-next-line
+  -v /path/to/data:/data:rw \
+  aica-runtime
+```
+
+</TabItem>
+<TabItem value="mac" label="macOS">
+
+```bash
+docker run -it --rm \
+  --privileged \
+  -p 8080:8080 -p 5000:5000 \
```

Comment:
```suggestion
  -p 8080:8080 -p 18000-18100:18000-18100/udp \
```
We shouldn't need `5000` as it's all mapped to `8080` but we need `18000-18100/udp` for WebRTC.